Description: Drop async-std and fs-err deps since they're not in Debian
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2022-10-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -123,17 +123,13 @@
     "time",
 ]
 all-impls = [
-    "async-std",
     "tokio",
     "os_pipe",
     "socket2",
     "mio",
-    "fs-err",
 ]
-async-std = ["io-lifetimes/async-std"]
 default = ["std"]
 fs = []
-fs-err = ["io-lifetimes/fs-err"]
 io_uring = [
     "linux-raw-sys",
     "fs",
